staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2673,6 +2673,10 @@ const (
 	// PodReasonSchedulerError reason in PodScheduled PodCondition means that some internal error happens
 	// during scheduling, for example due to nodeAffinity parsing errors.
 	PodReasonSchedulerError = "SchedulerError"
+
+	// TerminationByKubelet reason in DisruptionTarget pod condition indicates that the termination
+	// is initiated by kubelet
+	AlphaNoCompatGuaranteeTerminationByKubelet = "TerminationByKubelet"
```

Comment:
nit: I think we'll want to prefix with `PodReason` in the final name, like the other reasons here, so `AlphaNoCompatGuaranteePodReasonTerminationByKubelet`?

Reply:
Fine with me, actually I had such interim name then thought it's too long, but if others think it's ok, then I'm happy to move it back.